assets/src/application/ui/display/IconMenuItem/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Badge, Icon } from 'antd';
+import { BadgeProps } from 'antd/lib/badge';
+
+import './style.scss';
+
+interface IconMenuItemProps extends BadgeProps {
+	children?: React.ReactNode;
+	id?: string;
+	showClock?: boolean;
+}
+
+const IconMenuItem = ({ children, showClock, ...props }: IconMenuItemProps) => {
+	const count = showClock ? <Icon type='clock-circle' style={{ color: '#f5222d' }} /> : rest.count;
```

Comment:
Where does `rest.count` come from?

Reply:
oops, it was showing the error, but husky did allowed me to push, fixing now:
![image](https://user-images.githubusercontent.com/1477453/74449174-dc7c3a80-4e84-11ea-9382-b57d3d3a3783.png)